torch/csrc/jit/passes/to_batch.cpp

```diff
@@ -3,59 +3,543 @@
 
 namespace torch { namespace jit {
 
-std::unordered_map<std::string, std::shared_ptr<Graph>> ToBatch::batch_operator_table;
+std::unordered_map<std::string, std::vector<std::shared_ptr<Graph>>> ToBatch::batch_operator_table;
 
-void ToBatch::toBatch(Block* block, Block* res_block) {
-  // change inputs of a graph - expand tensor to {data, mask, dims}
-  auto size = block->inputs().size();
-  for(size_t i = 0; i < size; i++){
-    auto input = block->inputs()[i];
+std::shared_ptr<Graph> ToBatch::getBatchOperator(std::string name, int64_t input_num){
```

Comment:
Can you rename `input_num` to something like `num_inputs`? Right now it reads like "input number" which sounds like a number referring to a specific input, rather than "number of inputs"